src/AutoRest.CSharp/Mgmt/Output/ResourceOperation.cs

```diff
@@ -25,7 +25,7 @@ internal class ResourceOperation : TypeProvider
         private BuildContext<MgmtOutputLibrary> _context;
         private PagingMethod[]? _pagingMethods;
 
-        protected OperationGroup _operationGroup;
+        public OperationGroup _operationGroup { get; protected set; }
```

Comment:
We have added internal getter for OperationGroup in [Resource class](https://github.com/Azure/autorest.csharp/blob/f1fe1343f2a000fe3ed1d7fd7147d9643dd70f98/src/AutoRest.CSharp/Mgmt/Output/Resource.cs#L22). Can we do the same here?

Reply:
nit: `_operationGroup` -> `OperationGroup`